pkg/policy/groups/actions.go

```diff
@@ -150,9 +152,11 @@ func addDerivativeCNP(cnp *cilium_v2.CiliumNetworkPolicy) error {
 		if derivativeErr == nil {
 			break
 		}
+		metrics.PolicyImportErrors.Inc()
 		scopedLog.WithError(derivativeErr).Error("Cannot create derivative rule. Installing deny-all rule.")
 		statusErr := updateDerivativeStatus(cnp, derivativeCNP.ObjectMeta.Name, derivativeErr)
 		if statusErr != nil {
+			metrics.PolicyImportErrors.Inc()
```

Comment:
I don't think this one is necessary here because updating status doesn't affect whether there was an error importing the policy itself.